packages/frontend-2/components/settings/workspaces/members/TableHeader.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="flex flex-col-reverse md:justify-between md:flex-row md:gap-x-4">
+    <div class="relative w-full md:max-w-sm mt-6 md:mt-0">
+      <FormTextInput
+        name="search"
+        :custom-icon="MagnifyingGlassIcon"
+        color="foundation"
+        full-width
+        search
+        show-clear
+        :placeholder="searchPlaceholder"
+        class="rounded-md border border-outline-3"
+        v-bind="bind"
```

Comment:
Improved `useDebouncedTextInput` return to allow for way simpler/easier configuration of the FormTextInput component